src/vip/batch_address_test_tool/civic_info.clj

```diff
@@ -0,0 +1,30 @@
+(ns vip.batch-address-test-tool.civic-info
+  (:require [clojure.tools.logging :as log]
+            [clj-http.client :as client]
+            [turbovote.resource-config :refer [config]]
+            [clojure.data.json :as json]
+            [clojure.string :as str]))
+
+(defn response->polling-location
+  [response]
+  (when-let [address (get-in response [:body :pollingLocations 0 :address])]
+    (str/trim (str (:locationName address) ", "
+                   (:line1 address) ", "
+                   (:city address) ", "
+                   (:state address) " "
+                   (:zip address)))))
+
+(defn address->polling-location
+  [address]
+  (let [api-key (config [:civic-info-api-key])
+        url "https://www.googleapis.com/civicinfo/v2/voterinfo"
+        params {"address" address
+                "key" api-key
+                "electionId" 2000}
```

Comment:
Ah yeah, this is a leftover--basically, if you give the Civic Info API no election ID it should find whatever upcoming election is next associated with the address but if it can't find one it returns an error that your election ID is bad--this was in place because it worked with the addresses we were testing but you're right that it should be either 0 or nothing.